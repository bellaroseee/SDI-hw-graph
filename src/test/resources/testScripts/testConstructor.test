# Tests that a graph can be created

# Create a graph
CreateGraph graph1

# Create a graph that is already exist
CreateGraph graph1

# Create 2 graph
CreateGraph graph2
CreateGraph graph3

# Create 3 graph
CreateGraph graph4
CreateGraph graph5
CreateGraph graph6