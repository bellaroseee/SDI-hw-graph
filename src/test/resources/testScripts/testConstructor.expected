# Tests that a graph can be created

# Create a graph
created graph graph1

# Create a graph that is already exist


# Create 2 graph
created graph graph2
created graph graph3

# Create 3 graph
created graph graph4
created graph graph5
created graph graph6